src/engraving/style/styledef.cpp

```diff
@@ -483,7 +483,7 @@ const std::array<StyleDef::StyleValue, size_t(Sid::STYLES)> StyleDef::styleValue
     { Sid::HeaderToLineStartDistance,   "headerSlurTieDistance",   Spatium(1.0) },
 
     { Sid::tiePlacementSingleNote,   "tiePlacementSingleNote", TiePlacement::OUTSIDE },
-    { Sid::tiePlacementChord,        "tiePlacementChord",      TiePlacement::OUTSIDE },
+    { Sid::tiePlacementChord,        "tiePlacementChord",      TiePlacement::AUTO },
```

Comment:
As by @oktophonie 's comment, the new option is fine as an option but it shouldn't be the default, so this should stay to  OUTSIDE